pkgs/tools/backup/restic/default.nix

```diff
@@ -1,35 +1,35 @@
 {
-  stdenv,
   lib,
+  stdenv,
   buildGoModule,
   fetchFromGitHub,
   installShellFiles,
   makeWrapper,
-  nixosTests,
   rclone,
+  nixosTests,
```

Comment:
Why unnecessarily move this?

Reply:
Just to adhere to chronological usage ( bundle input for tests at the bottom ) . I can revert this change.